### PR TITLE
🦄 implement grpc gateway

### PR DESCRIPTION
**Pull request by nycae**

### CODE REVIEW

File: buf.gen.yaml [hunk 1]
```diff
@@ -7,20 +7,28 @@ plugins:
     out: gen/python
     path: protoc-gen-grpclib_python
 
-  - remote: buf.build/library/plugins/go:v1.27.1-1
```
Comment (magmax) [inline buf.gen.yaml:10]: why to reorder? Could you keep remote url instead of name?

File: buf.gen.yaml [hunk 2]
```diff
@@ -7,20 +7,28 @@ plugins:
...
-    out: gen/go
```
Comment (magmax) [inline buf.gen.yaml:11]: why to remove `standalone`?

File: gateway/pkg/sekura/v1/sekura.pb.go [hunk 3]
```diff
@@ -0,0 +1,1396 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```
Comment (magmax) [inline gateway/pkg/sekura/v1/sekura.pb.go:1]: Please, do not commit generated code.

File: gateway/pkg/sekura/v1/sekura.pb.gw.go [hunk 4]
```diff
@@ -0,0 +1,845 @@
+// Code generated by protoc-gen-grpc-gateway. DO NOT EDIT.
```
Comment (magmax) [inline gateway/pkg/sekura/v1/sekura.pb.gw.go:1]: ditto

File: gateway/pkg/sekura/v1/sekura_grpc.pb.go [hunk 5]
```diff
@@ -0,0 +1,445 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```
Comment (magmax) [inline gateway/pkg/sekura/v1/sekura_grpc.pb.go:1]: ditto

File: sekura/v1/sekura.proto [hunk 6]
```diff
@@ -4,7 +4,7 @@ syntax = "proto3";
 //
 // API description for Sekura services.
 package sekura.v1;
-option go_package = "github.com/sekuradev/sekura/gen/go/sekura/v1";
+option go_package = "github.com/sekuradev/api/gateway/pkg/sekura/v1";
```
Comment (magmax) [inline sekura/v1/sekura.proto:7]: why to change this? From the point of view of an app using gRPC, it won't use a gateway.

File: buf.gen.yaml (hunk 1 above)
Comment (nycae) [reply buf.gen.yaml:10]: All the changes were made by running buf generate.

File: sekura/v1/sekura.proto (hunk 6 above)
Comment (nycae) [reply sekura/v1/sekura.proto:7]: I need this in order for protoc to generate the files in the correct directory

**Comment by magmax** [comment]
Please do you mind removing generated files from PR and solve conflicts?
After that I will merge your PR.

**Pull request merged**